wiki/Help_centre/Beatmapping/it.md

```diff
@@ -0,0 +1,51 @@
+---
+tags: 
+  - upload
+  - delete
+  - beatmap
+  - slot
+  - editor
+  - status
+  - warning
+  - caricare
+  - attenzione
+  - 
+---
+
+# Beatmapping e l'Editor
+
+*Pagina principale: [Centro di Supporto](/wiki/it/Help_centre)*
+
+Questa sezione è dedicata ai problemi riguardo il Beatmap Editor e generalmente beatmapping.  
+
+## Come carico/aggiorno la mia mappa? {#upload-beatmap}
+
+**Puoi farlo tramite l'editor in gioco.**
+
+Per caricare la tua beatmap, vai nell'editor e clicca `File` > `Carica Beatmap`.
+
+Questo renderà la tua beatmap disponibile da scaricare agli altri giocatori sul sito web. La stessa procedura vale nel caso si voglia aggiornare una mappa già caricata.
+
+Ogni utente ha un numero limitato di mappe che possono caricare. Lasciare la mappa inattiva per un mese (28 giorni), senza aggiornarla, verrà mandata nelle [Beatmap Abbandonate](/wiki/Beatmap/Category#graveyard). Per risvegliare una beatmap da quelle abbandonate a quelle in attesa o WIP, basta aggiornarla.
```

Comment:
`/wiki/Beatmap/Category#graveyard`
This will take us to the english version of the page. The linked page as not been translated to italian yet, so we might need to make a new PR to fix it when it's out.